src/gt4py/next/program_processors/runners/dace/workflow/translation.py

```diff
@@ -88,6 +88,11 @@ def generate_sdfg(
         if not self.disable_itir_transforms:
             ir = itir_transforms.apply_fieldview_transforms(ir, offset_provider=offset_provider)
         offset_provider_type = common.offset_provider_to_type(offset_provider)
+
+        # do not store transformation history in SDFG
+        dace_config_store_history = dace.Config.get("store_history")
+        dace.Config.set("store_history", value=False)
```

Comment:
```suggestion
with dace.config.set_temporary("store_history", value=False):
```
Can you use the context handler here?